docs/DG_Diagrams/ModelComponent.puml

```diff
@@ -0,0 +1,26 @@
+@startuml
+
+Rectangle Model {
+Rectangle bus {
+enum BusStops {
+PGP
+KENTRIDGEMRTSTATION
+...
+COLLEGEGREEN
+OEITIONGHAM
+}
+Bus ->"*" BusStops
+BusData ->"*" Bus
+BusInfo ->"*" Bus
+BusInfo ..> BusData
+}
+Rectangle favorite {
+FavList->"*"Fav
+}
+Rectangle foodoptions {
+FoodPlace ->"*" DiningOptions
+FoodPlacesData ..>FoodPlace
+}
+}
+
```

Comment:
![Screenshot_3](https://user-images.githubusercontent.com/59549150/97514831-a8bd5280-19ca-11eb-9230-9cda5c173dc6.png)

Can consider including the visibility and type for the attributes if they are stated inside the box for the classes, because not sure if it is optional.